frontend/app/components/ToolCard.tsx

```diff
@@ -80,7 +66,43 @@ export default function ToolCard (props: ToolCardProps) {
           <DataElement name='Description'>
             <span dangerouslySetInnerHTML={{ __html: tool.long_description }} />
           </DataElement>
-          <ContainedCardMedia {...mainImageProps} />
+          <CardMedia
+            component='img'
+            height={150}
+            alt={tool.main_image_alt_text ?? 'Image of tool in use'}
+            image={tool.main_image ?? ''}
+            sx={{ marginBottom: 2, objectFit: 'contain' }}
+          />
+          <CardActions>
+            <Button onClick={() => setScreenshotDialogOpen(true)} startIcon={<AddBox />}>
+              Enlarge Screenshot
+            </Button>
+          </CardActions>
+          <Dialog
+            fullWidth
+            maxWidth='xl'
+            open={screenshotDialogOpen}
+            onClose={() => setScreenshotDialogOpen(false)}
+            aria-labelledby='main-image-dialog-title'
+          >
+            <DialogTitle id='main-image-dialog-title'>Screenshot of {tool.name}</DialogTitle>
+            <DialogContent>
+              <DialogImg
+                tabIndex={0}
+                src={tool.main_image ?? ''}
+                alt={tool.main_image_alt_text ?? 'Image of tool in use'}
+              />
+            </DialogContent>
+            <DialogActions>
+              <Button
+                aria-label='Close screenshot dialog'
+                onClick={() => setScreenshotDialogOpen(false)}
+                startIcon={<CloseIcon />}
+              >
+                Close
+              </Button>
+            </DialogActions>
+          </Dialog>
```

Comment:
A thought: I think it's reasonable to expect that every tool that we post information about has a `logo_image`, but it's possible we may want to go live with tools without a screenshot. Should we make all of this conditional, hiding it when `tool.main_logo` is not defined? Thoughts? This could be done now or in a separate issue.

Reply:
That's good idea. As of now, neither the "Logo image" or the "Main image" is of required field, so either could be null.

Is it possible to also make the image itself clickable, so that once the user clicks on the main image (if exists), the dialog window will be shown, the same user experience as if the clicked on the "Enlarge Screenshot" button.